SPHINXsys/src/for_3D_build/particle_generator/particle_generator_network.cpp

```diff
@@ -45,7 +45,7 @@ namespace SPH
 		return downgrad - upgrad;
 	}
 	//=================================================================================================//
-	Vecd ParticleGeneratorNetwork::creatATentativeNewBranchVecd(Vecd init_point, Vecd dir)
+	Vecd ParticleGeneratorNetwork::creatATentativeNewBranchPoint(Vecd init_point, Vecd dir)
```

Comment:
there is a typo:: create

Reply:
I will correct the typo in bitbucket repo, it will be updated in the next merge.